Nome: Gustavo Nakaura de Carvalho |||| Ra: 816253667

1 Faça um Programa que peça dois números e imprima o maior deles.

In [1]:
n1  = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número:  "))

if n1 > n2:
    print("O maior número é : {}".format(n1))
else:
    print("O maior número é : {}".format(n2))

Digite o primeiro número: 5
Digite o segundo número:  7
O maior número é : 7


2 Faça um Programa que verifique se uma letra digitada é vogal ou consoante.

In [2]:
string = input("Digite uma letra: ")
if string in ['a','e','i','o','u']:
    print('É vogal!')
else:
    print('É consoante!')

Digite uma letra: p
É consoante!


3 Faça um programa para a leitura de duas notas parciais de um aluno. O programa
deve calcular a média alcançada por aluno e apresentar:
- A mensagem "Aprovado", se a média alcançada for maior ou igual a sete;
- A mensagem "Reprovado", se a média for menor do que sete;
- A mensagem "Aprovado com Distinção", se a média for igual a dez.

In [4]:
nota1 = int(input('Digite sua nota: '))
nota2 = int(input('Digite sua 2° nota: '))

media = (nota1 + nota2) / 2

if media >= 7 and nota1 < 10:     
    print ('Parabéns! Você foi aprovado!')
elif media >= 10:
    print ('Você foi aprovado com distinção!')    
else:
    print ('Infelizmente você foi reprovado!')

Digite sua nota: 9
Digite sua 2° nota: 7
Parabéns! Você foi aprovado!


4 Faça um Programa que leia três números e mostre-os em ordem decrescente.

In [5]:
n1 = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número: "))
n3 = int(input("Digite o terceiro número: "))

if n1 < n2 and n1 < n3 and n3 < n2:
    print(n2,n3,n1)
elif n2 < n1 and n2 < n3 and n1 < n3: 
    print(n3,n1,n2)
elif n3 < n1 and n3 < n2 and n1 < n2:     
    print(n2,n1,n3)
elif n1<n2 and n2<n3 and n1<n3:
    print(n3,n2,n1)
elif n2<n3 and n3<n1 and n3<n1:
    print(n1,n3,n2)
else:
    print(n1,n2,n3)

Digite o primeiro número: 7
Digite o segundo número: 5
Digite o terceiro número: 3
7 5 3


5 As Organizações Tabajara resolveram dar um aumento de salário aos seus
colaboradores e lhe contrataram para desenvolver o programa que calculará os
reajustes.
Faça um programa que recebe o salário de um colaborador e o reajuste segundo o
seguinte critério, baseado no salário atual:
- salários até R$ 280,00 (incluindo) : aumento de 20%
- salários entre R$ 280,00 e R$ 700,00 : aumento de 15%
- salários entre R$ 700,00 e R$ 1500,00 : aumento de 10%
- salários de R$ 1500,00 em diante : aumento de 5% Após o aumento ser realizado,
informe na tela:
- o salário antes do reajuste;
- o percentual de aumento aplicado;
- o valor do aumento;
- o novo salário, após o aumento.

In [9]:
salario = float(input("Digite seu sálario: "))

if salario <= 280.00:
    plus = salario * 20/100
elif salario > 280.00 and salario < 700.00:
    plus = salario * 15/100
elif salario > 700.00 and salario < 1500.00:
    plus = salario * 10/100
elif salario > 1500.00:
    plus = salario * 5/100

novo = salario + plus    
print("Salário antes do reajuste: {} R$\nPercentual de aumento aplicado: {}\nNovo sálario: {} R$".format(salario,plus,novo))

Digite seu sálario: 4500
Salário antes do reajuste: 4500.0 R$
Percentual de aumento aplicado: 225.0
Novo sálario: 4725.0 R$


6 Faça um Programa que leia um número e exiba o dia correspondente da semana. (1-Domingo, 2- Segunda, etc.), se digitar outro valor deve aparecer valor inválido

In [10]:
dia = int(input("Digite o dia da semana "))

if dia == 1:
    print("O dia da semana é: Domingo")
elif dia == 2:
    print("O dia da semana é: Segunda")
elif dia == 3:
    print("O dia da semana é: Terça")
elif dia == 4:
    print("O dia da semana é: Quarta")
elif dia == 5:
    print("O dia da semana é: Quinta")
elif dia == 6:
    print("O dia da semana é: Sexta")
elif dia == 7:
    print("O dia da semana é: Sábado")
else:
    print ('INVALIDO')

Digite o dia da semana 8
INVALIDO


7 Faça um programa que lê as duas notas parciais obtidas por um aluno numa disciplina ao longo de um semestre, e calcule a sua média. A atribuição de conceitos obedece à tabela abaixo: Média de Aproveitamento Conceito Entre 9.0 e 10.0 A Entre 7.5 e 9.0 B Entre 6.0 e 7.5 C Entre 4.0 e 6.0 D Entre 4.0 e zero E O algoritmo deve mostrar na tela as notas, a média, o conceito correspondente e a mensagem “APROVADO” se o conceito for A, B ou C ou “REPROVADO” se o conceito for D ou E.

In [11]:
n1 = float(input("Digite a primeira nota: "))
n2 = float(input("Digite a segunda nota: "))

media = (n1 + n2)/2

if media >= 9 and media <= 10:
    print("APROVADO -> A")
elif media >= 7.5 and media < 9:
    print("APROVADO -> B")
elif media >= 6 and media < 7.5:
    print("APROVADO -> C")
elif media >= 4 and media < 6:
    print("REPROVADO -> D")
else:
    print("REPROVADO -> E")

Digite a primeira nota: 10
Digite a segunda nota: 5
APROVADO -> B


8 Faça um Programa que peça os 3 lados de um triângulo. O programa deverá
informar se os valores podem ser um triângulo. Indique, caso os lados formem um
triângulo, se o mesmo é: equilátero, isósceles ou escaleno.
Dicas:
- Três lados formam um triângulo quando a soma de quaisquer dois lados for maior
que o terceiro;
- Triângulo Equilátero: três lados iguais;
- Triângulo Isósceles: quaisquer dois lados iguais;
- Triângulo Escaleno: três lados diferentes;

In [12]:
l1 = input('Digite o tamanho do primeiro lado: ')
l2 = input('Digite o segundo lado: ')
l3 = input('Digite o terceiro lado: ')

if l1 + l2 > l3:
    if l1 == l2 and l1 == l3:
        print('Triângulo Equilátero')
    elif l1 == l2 or l2 == l3 or l1 == l3:
        print('Triângulo Isósceles')
    elif l1 != l2 and l3 or l2 != l1 and l3 or l1 != l3:
        print('Triângulo Escaleno')
else:
    print('Não é um triângulo')

Digite o tamanho do primeiro lado: 7
Digite o segundo lado: 8
Digite o terceiro lado: 3
Triângulo Escaleno


 9 Faça um programa que calcule as raízes de uma equação do segundo grau, na
forma ax2 + bx + c. O programa deverá pedir os valores de a, b e c e fazer as
consistências, informando ao usuário nas seguintes situações:
- Se o usuário informar o valor de A igual a zero, a equação não é do segundo grau e
o programa não deve fazer pedir os demais valores, sendo encerrado;
- Se o delta calculado for negativo, a equação não possui raizes reais. Informe ao
usuário e encerre o programa;
- Se o delta calculado for igual a zero a equação possui apenas uma raiz real;
informe-a ao usuário;
- Se o delta for positivo, a equação possui duas raiz reais; informe-as ao usuário;

In [13]:
import math

print("Digite os termos da equacao a, b e c da equacao ax^2 + bx + c")
a = float(input("Digite o termo a ---> "))
if a == 0:
    print("nao e uma equacao de segundo grau")
else:
    b = float(input("Digite o termo b  ---> "))
    c = float(input("Digite o termo c ---> "))
    delta = (math.pow(b,2) - (4*a*c))
if delta < 0:
    print("Delta = {} a equacao nao possui raizes reais".format(delta))
if delta == 0:
    print("Delta = {} a equacao possui uma raiz".format(delta))
    raiz = ((-1)*b + math.sqrt(delta))/(2*a)
    print("Raiz da equacao = {}".format(raiz))
if delta > 0:
    print("Delta = {} a equacao possui duas raizes".format(delta))
    raiz1 = ((-1)*b + math.sqrt(delta))/(2*a)
    raiz2 = ((-1)*b - math.sqrt(delta))/(2*a)
    print("Raiz 1 da equacao = ".format(raiz1))
    print("Raiz 2 da equacao = ".format(raiz2))

Digite os termos da equacao a, b e c da equacao ax^2 + bx + c
Digite o termo a ---> 4
Digite o termo b  ---> 3
Digite o termo c ---> 5
Delta = -71.0 a equacao nao possui raizes reais


10 Faça um Programa para um caixa eletrônico. O programa deverá perguntar ao
usuário a valor do saque e depois informar quantas notas de cada valor serão
fornecidas. As notas disponíveis serão as de 1, 5, 10, 50 e 100 reais. O valor mínimo é
de 10 reais e o máximo de 600 reais. O programa não deve se preocupar com a
quantidade de notas existentes na máquina.
- Exemplo 1: Para sacar a quantia de 256 reais, o programa fornece duas notas de
100, uma nota de 50, uma nota de 5 e uma nota de 1;
- Exemplo 2: Para sacar a quantia de 399 reais, o programa fornece três notas de
100, uma nota de 50, quatro notas de 10, uma nota de 5 e quatro notas de 1.

In [15]:
saque = int(input("Digite o valor do saque: "))

if 10 <= saque <= 600:
    notasdeCem = saque // 100
    saque = saque % 100

    notasdeCinquenta = saque // 50
    saque = saque % 50

    notasdeDez = saque // 10
    saque = saque % 10

    notasdeCinco = saque // 5
    saque = saque % 5

    notasdeUm = saque // 1

    if notasdeCem > 0:
        print(notasdeCem, "notas de R$ 100")
    if notasdeCinquenta > 0:
        print(notasdeCinquenta, "notas de R$ 50")
    if notasdeDez > 0:
        print(notasdeDez, "notas de R$ 10")
    if notasdeCinco > 0:
        print(notasdeCinco, "notas de R$ 5")
    if notasdeUm > 0:
        print(notasdeUm, "notas de R$ 1")
              
else:
    print("Nao é possivel fazer o saque")

Digite o valor do saque: 50
1 notas de R$ 50


11 Faça um programa que faça 5 perguntas para uma pessoa sobre um crime. As
perguntas são:
"Telefonou para a vítima?"
"Esteve no local do crime?"
"Mora perto da vítima?"
"Devia para a vítima?"
"Já trabalhou com a vítima?"
O programa deve no final emitir uma classificação sobre a participação da pessoa no
crime. Se a pessoa responder positivamente a 2 questões ela deve ser classificada
como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino". Caso contrário,
ele será classificado como "Inocente".

In [16]:
print("Lembre-se 1 para sim e 0 para não")

p1  = int(input("Telefonou para a vítima? "))
p2 = int(input("Esteve no local do crime? "))
p3 = int(input("Mora perto da vítima? "))
p4 = int(input("Devia para a vítima? "))
p5 = int(input("Já trabalhou com a vítima? "))

res = p1 + p2 + p3+ p4 + p5

if res == 2:
    print("Suspeita")
elif res == 3 or res == 4:
    print("Cúmplice")
elif res == 5:
    print("Assassino")
else:
    print("Inocente")

Lembre-se 1 para sim e 0 para não
Telefonou para a vítima? 1
Esteve no local do crime? 1
Mora perto da vítima? 1
Devia para a vítima? 0
Já trabalhou com a vítima? 1
Cúmplice


12 Uma fruteira está vendendo frutas com a seguinte tabela de preços:
Até 5 Kg Acima de 5 Kg
Morango R$ 2,50 por Kg R$ 2,20 por Kg
Maçã R$ 1,80 por Kg R$ 1,50 por Kg
Se o cliente comprar mais de 8 Kg em frutas ou o valor total da compra ultrapassar
R$ 25,00, receberá ainda um desconto de 10% sobre este total. Escreva um
algoritmo para ler a quantidade (em Kg) de morangos e a quantidade (em Kg) de
maças adquiridas e escreva o valor a ser pago pelo cliente.

In [ ]:
morango = float(input("Digite a quantidade (em kg) de morangos "))
maca   = float(input("Digite a quantidade (em kg) de maçãs "))

if morango < 5 and maca < 5:
    quantidade = morango + maca
    precoMorango = morango * 2.50
    precoMaca = maca * 1.50
    valor = precoMorango + precoMaca
    if quantidade > 8 or valor > 25:
        des = valor * 10/100
        novo = valor - des
        print("Valor a ser pago R${}".format(novo))
    else:
        print("Valor a ser pago R${}".format(valor))
else:
    print("Quantidade acima do permitido.")

13 Faça um programa que peça uma nota, entre zero e dez. Mostre uma mensagem
caso o valor seja inválido e continue pedindo até que o usuário informe um valor
válido.

In [ ]:
n = 0
while n >= 0 and n <= 10:
    n = int(input("Digite uma nota entre 0 e 10:  "))

else:
    print("Acabou!")

14 Faça um programa que leia e valide as seguintes informações: Nome: maior que 3 caracteres; Idade: entre 0 e 150; Salário: maior que zero; Sexo: 'f' ou 'm'; Estado Civil: 's', 'c', 'v', 'd';

In [ ]:
nome = input("Digite seu nome: (digite mais que 3 caracteres)")
i = int(input("Digite sua idade: (entre 0 e 150)"))
s = float(input("Digite seu sálario: (maior que zero) "))
sex = input("Digite seu sexo: ('f' ou 'm') ")
es = input("Digite seu estado civil: ('s', 'c', 'v', 'd')")

if len(nome) > 3:
    print("Nome: {}".format(nome))
else:
    nome = input("Digite seu nome novamente ")
    print("Nome: {}".format(nome))
if s > 0:
    print("Sálario: {} R$".format(s))
else:
    s = input("Digite seu sálario novamente ")
if sex == 'f' or 'm':
    if sex == 'f':
        print("Sexo Feminino")
    else:
        print("Sexo Masculino")
else:
    sex = input("Digite seu sexo novamente")
if es == 's' or 'c' or 'v' or 'd':
    print("Estado civil: {}".format(es))
else:
    es = input("Digite seu estado civil novamente")
if i >= 0 and i <=150:
    print("Idade: {}".format(i))
else:
    i = input("Digite sua idade *novamente")

15 Faça um programa que leia 5 números e informe o maior número.

In [ ]:
n1 = int(input("Digite 1 número: "))
n2 = int(input("Digite 2 número: "))
n3 = int(input("Digite 3 número: "))
n4 = int(input("Digite 4 número: "))
n5 = int(input("Digite 5 número: "))

if n1 > n2 and n > n3 and n1 > n4 and n1 > n5:
    print(n)
elif n2 > n3 and n2 > n4 and n2 > n5 and n2 > n:
    print(n2)
elif n3 > n2 and n3 > n1 and n3 > n4 and n3 > n5:
    print(n3)
elif n4 > n2 and n4 > n1 and n4 > n3 and n4 > n5:
    print(n4)
else:
    print(n5)

16 Faça um programa que leia 5 números e informe a soma e a média dos números

In [ ]:
n1 = int(input("Digite 1 número "))
n2 = int(input("Digite 2 número "))
n3 = int(input("Digite 3 número "))
n4 = int(input("Digite 4 número "))
n5 = int(input("Digite 5 número "))

soma = n1 + n2 + n3 +n4 +n5
media = soma/5

print("Soma: {} Média:{}".format(soma,media))

17  Faça um programa que imprima na tela apenas os números ímpares entre 1 e 50

In [ ]:
i = 0
impar = ""
while i <= 49:
    i += 1
    if i%2 != 0:
        impar += str(i) + " "
        
else:
    print("Impares: {}".format(impar))

18 Desenvolva um gerador de tabuada, capaz de gerar a tabuada de qualquer número inteiro entre 1 a 10. O usuário deve informar de qual numero ele deseja ver a tabuada. A saída deve ser conforme o exemplo abaixo: Tabuada de 5: 5 X 1 = 5 5 X 2 = 10 ... 5 X 10 = 50

In [ ]:
n1= int(input("Digite o número da tabuada "))
mult = 0
if n1 >= 1 and n <= 10:
    c = 1
    while c <= 10:
        mult = n1 * c
        print("{} X {} = {}".format(n1,c,mult))
        c += 1
    else:
        print("Acabou")
else:
    print("Algo errado")

19 A série de Fibonacci é formada pela seqüência 1,1,2,3,5,8,13,21,34,55,... Faça um programa capaz de gerar a série até o n−ésimo termo.

In [19]:
a, b = 0, 1
n1 = 99
while a < n1:
    print(a, end=' ')
    a, b = b, a+b
    print()
else:
    print("Acabou")

0 
1 
1 
2 
3 
5 
8 
13 
21 
34 
55 
89 
Acabou


20 Faça um programa que calcule o fatorial de um número inteiro fornecido pelo usuário. Ex.: 5!=5.4.3.2.1=120

In [20]:
num = int(input("Fatorial de: ") )

resultado=1
count=1
mostrar = ""
while count <= num:
    resultado *= count
    mostrar += str(count) + "."
    count += 1

print("{}!= {} = {}".format(num,mostrar[::-1],resultado))

Fatorial de: 4
4!= .4.3.2.1 = 24


21 O Sr. Manoel Joaquim possui uma grande loja de artigos de R$ 1,99, com cerca
de 10 caixas. Para agilizar o cálculo de quanto cada cliente deve pagar ele
desenvolveu um tabela que contém o número de itens que o cliente comprou e ao
lado o valor da conta. Desta forma a atendente do caixa precisa apenas contar
quantos itens o cliente está levando e olhar na tabela de preços. Você foi contratado
para desenvolver o programa que monta esta tabela de preços, que conterá os
preços de 1 até 50 produtos, conforme o exemplo abaixo:
Lojas Quase Dois - Tabela de preços
1 - R$ 1.99
2 - R$ 3.98
...
50 - R$ 99.50

In [17]:
itens = int(input("Quantos itens o cliente tem? "))
i = 0
mostrar = ""
if itens <= 50:
    while i <= itens:
        p = float(input("Digite o preço do produto "))
        mostrar += str(i) + " - R$ " + str(p) + "\n"
        i += 1
    else:
        print(mostrar)
else:
    print("Quantida acima de 50")

Quantos itens o cliente tem? 6
Digite o preço do produto 5
Digite o preço do produto 5
Digite o preço do produto 30
Digite o preço do produto 50
Digite o preço do produto 40
Digite o preço do produto 90
Digite o preço do produto 60
0 - R$ 5.0
1 - R$ 5.0
2 - R$ 30.0
3 - R$ 50.0
4 - R$ 40.0
5 - R$ 90.0
6 - R$ 60.0



22 O Sr. Manoel Joaquim acaba de adquirir uma panificadora e pretende implantar
a metodologia da tabelinha, que já é um sucesso na sua loja de 1,99. Você foi
contratado para desenvolver o programa que monta a tabela de preços de pães, de
1 até 50 pães, a partir do preço do pão informado pelo usuário, conforme o exemplo
abaixo:
Preço do pão: R$ 0.18
Panificadora Pão de Ontem - Tabela de preços
1 - R$ 0.18
2 - R$ 0.36
...
50 - R$ 9.00

In [22]:
itens = int(input("Quantos pães o cliente comprou? "))
i = 0
mostrar = ""
if itens <= 50:
    while i <= itens:
        p = float(input("Digite o preço "))
        mostrar += str(i) + " - R$ " + str(p) + "\n"
        i += 1
    else:
        print(mostrar)
else:
    print("Quantida acima de 50")

Quantos pães o cliente comprou? 5
Digite o preço 1.20
Digite o preço 2.20
Digite o preço 3.40
Digite o preço 5.60
Digite o preço 1.00
Digite o preço 1.00
0 - R$ 1.2
1 - R$ 2.2
2 - R$ 3.4
3 - R$ 5.6
4 - R$ 1.0
5 - R$ 1.0



23 O Sr. Manoel Joaquim expandiu seus negócios para além dos negócios de 1,99 e
agora possui uma loja de conveniências. Faça um programa que implemente uma
caixa registradora rudimentar. O programa deverá receber um número
desconhecido de valores referentes aos preços das mercadorias. Um valor zero deve
ser informado pelo operador para indicar o final da compra. O programa deve então
mostrar o total da compra e perguntar o valor em dinheiro que o cliente forneceu,
para então calcular e mostrar o valor do troco. Após esta operação, o programa
deverá voltar ao ponto inicial, para registrar a próxima compra. A saída deve ser
conforme o exemplo abaixo:
Lojas Tabajara
Produto 1: R$ 2.20
Produto 2: R$ 5.80
Produto 3: R$ 0
Total: R$ 9.00
Dinheiro: R$ 20.00
Troco: R$ 11.00
...

In [2]:
print("Bem-vindo a Loja Gustavo Tabajara  - Lembre-se: para sair digite 0")
p = 1
i = 1
mostrar = ""
total = 0
while p != 0:
    p = float(input("Digite o preço do produto "))
    mostrar += "Produto: " + str(i) + ": R$ " + str(p) + "\n"
    total = total + p 
    i += 1
else:
    d = float(input("Digite o dinheiro fornecido "))
    troco = d - total
    print("{}\n Dinheiro: {} R$\nTroco: {} R$".format(mostrar,d,troco))

Bem-vindo a Loja Gustavo Tabajara  - Lembre-se: para sair digite 0
Digite o preço do produto 45
Digite o preço do produto 20
Digite o preço do produto 41
Digite o preço do produto 60
Digite o preço do produto 10
Digite o preço do produto 0
Digite o dinheiro fornecido 280
Produto: 1: R$ 45.0
Produto: 2: R$ 20.0
Produto: 3: R$ 41.0
Produto: 4: R$ 60.0
Produto: 5: R$ 10.0
Produto: 6: R$ 0.0

 Dinheiro: 280.0 R$
Troco: 104.0 R$


24 Foi feita uma estatística em cinco cidades brasileiras para coletar dados sobre
acidentes de trânsito. Foram obtidos os seguintes dados:
- Código da cidade;
- Número de veículos de passeio (em 1999);
- Número de acidentes de trânsito com vítimas (em 1999). Deseja-se saber:
- Qual o maior e menor índice de acidentes de transito e a que cidade pertence;
- Qual a média de veículos nas cinco cidades juntas;
- Qual a média de acidentes de trânsito nas cidades com menos de 2.000 veículos de
passeio.

In [6]:
i = 0
acidMaior = 0
acidMenor = 2000
total = 0
totalMenor= 0
div = 0
while i < 5:
    cod = int(input("Digite o código da cidade "))
    num_v = int(input("Digite o Número de veículos de passeio "))
    num_a = int(input("Digite o Número de acidentes de trânsito com vítimas "))
    print("\n")
    
    if num_a > acidMaior:
        acidMaior = num_a
        m = cod
    if num_a < acidMenor:
        acidMenor = num_a
        m2= cod
    
    if num_v < 2000:
        totalMenor += num_v
        div += 1
    else:
        total += num_v
    
    i += 1
else:
    mediaMenor = totalMenor / div
    media = total / 5
    print("Código: {}° Maior: {}\nCódigo: {}° Menor: {}\nMedia entre as cidades: {}\nMedia acidentes acima de 2.000: {}".format(
        m,acidMaior,m2,acidMenor,media,mediaMenor))

Digite o código da cidade 11
Digite o Número de veículos de passeio 520
Digite o Número de acidentes de trânsito com vítimas 15


Digite o código da cidade 15
Digite o Número de veículos de passeio 650
Digite o Número de acidentes de trânsito com vítimas 25


Digite o código da cidade 16
Digite o Número de veículos de passeio 250
Digite o Número de acidentes de trânsito com vítimas 147


Digite o código da cidade 13
Digite o Número de veículos de passeio 258
Digite o Número de acidentes de trânsito com vítimas 20


Digite o código da cidade 10
Digite o Número de veículos de passeio 256
Digite o Número de acidentes de trânsito com vítimas 85


Código: 16° Maior: 147
Código: 11° Menor: 15
Media entre as cidades: 0.0
Media acidentes acima de 2.000: 386.8


25 Faça um programa que receba o valor de uma dívida e mostre uma tabela com
os seguintes dados: valor da dívida, valor dos juros, quantidade de parcelas e valor
da parcela.
Os juros e a quantidade de parcelas seguem a tabela abaixo:


In [ ]:
vd = float(input("Digite o valor da dívida "))
qp = float(input("Digite a quantidade de parcelas "))

if qp == 1:
    vj = 0
    vp = vd
elif qp == 3:
    vj = 10
    vp = vd * vj/100
elif qp == 6:
    vj = 15
    vp = vd * vj/100
elif qp == 9:
    vj = 20
    vp = vd * vj/100
else:
    vj = 25
    vp = vd * vj/100

print("Valor da Dívida: R$ {} Valor dos Juros: R$ {} Quantidade de Parcelas {} Valor da Parcela R$ {}".
     format(vd,vj,qp,vp))

26 Faça um programa que mostre os n termos da Série a seguir: S = 1/1 + 2/3 + 3/5 + 4/7 + 5/9 + ... + n/m. Imprima no final a soma da série.

In [ ]:
n = int(input("entre com o valor de n: "))
s = 1/n + (2/(n-1)) + (3/(n-2)) + ((n-1)/2) + n
print(s)